tensorflow_datasets/testing/dataset_builder_testing.py

```diff
@@ -137,7 +137,10 @@ class DatasetBuilderTestCase(parameterized.TestCase, test_utils.SubTestCase):
   EXAMPLE_DIR = None
   OVERLAPPING_SPLITS = []
   MOCK_OUT_FORBIDDEN_OS_FUNCTIONS = True
-
+  SKIP_CHECKSUMS = False
+  URLS = []
+  CHECKSUMS_PATH = None 
```

Comment:
The checksum path should be automatically look-up, by checking all the registered checksum paths. This shouldn't be a property.